src/js/utils/parser.js

```diff
@@ -142,9 +142,9 @@ define([
      * Determine the adaptive type
      */
     parser.adaptiveType = function(duration) {
-        if (duration !== -1) {
+        if (duration !== 0) {
```

Comment:
No. line 154 returns "VOD". The -1 behavior was for jw6. It should have been changed when providers were modified to return 0 instead of -1 as the default unknown duration.